diff --git a/src/components/InternalPortal.tsx b/src/components/InternalPortal.tsx
index 351572a..8092e2d 100644
--- a/src/components/InternalPortal.tsx
+++ b/src/components/InternalPortal.tsx
@@ -945,7 +945,7 @@ export default function InternalPortal({
   const [intakeRows, setIntakeRows] = useState<any>([]);
   const [intakeSearchTerm, setIntakeSearchTerm] = useState<any>("");
   const [inventoryCategoryFilter, setInventoryCategoryFilter] =
-    useState<any>("");
+    useState<any>("Todos");
   const [isCatalogOpen, setIsCatalogOpen] = useState<any>(false);
   const [labBenchName, setLabBenchName] = useState<any>("");
   const [labBenchSuccessMessage, setLabBenchSuccessMessage] = useState<any>("");
